linux/shim/service.go

```diff
@@ -211,6 +212,7 @@ func (s *Service) Resume(ctx context.Context, r *shimapi.ResumeRequest) (*google
 }
 
 func (s *Service) Exit(ctx context.Context, r *shimapi.ExitRequest) (*google_protobuf.Empty, error) {
+	unix.Unmount(filepath.Join(s.path, "rootfs"), 0)
```

Comment:
May want to have it in a linux specific file, from what I've seen of @justincormack changes this is only going to be needed on `linux`